### PR TITLE
Added .pokernow

### PR DESCRIPTION
**Pull request by lovetoburnswhen**
- `.pokernow plot` + attach the log file

### CODE REVIEW

File: ext/maga.py [hunk 1]
```diff
@@ -1,10 +0,0 @@
-async def maga(message):
```
Comment (blerud) [inline ext/maga.py:1]: why is this deleted

File: util/config.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
-vac_file: str = 'vac.json'
-csgo_id: int = 454124197048745996
-schedule_file: str = 'schedule.json'
+PREFIX: str = '.'
+DEFAULT_GUILD: int = 90647915084677120
+DEFAULT_CHANNEL: int = 191661931436703744
+VAC_FILE: str = 'vac.json'
+CSGO_ID: int = 454124197048745996
+SCHEDULE_FILE: str = 'schedule.json'
+POKERNOW_LOG_DIR: str = './pokernowlogs/'  # Archiving for posterity (rankings who lost money to whom etc)
```
Comment (blerud) [inline util/config.py:7]: dont need the comment

File: ext/pokernow.py [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+        ```'''
+    )
+
+    # r'\"(\w+(?:\s\w+)?) \@ (\w+)\" (joined|quits) the game with a stack of (\d+\.\d\d)'
+    # df = pd.read_csv('poker_now_log.csv', parse_dates=True, infer_datetime_format=True)
+    # username, uid, event_type, quantity
+    # entry_exits = df.entry.str.extract(
+    #     r'"(.+(?:.+)?)\s@\s(.+)"+ (joined|quits) the game with a stack of (\d+\.\d\d)'
+    # ).dropna().iloc[::-1]
+    # losing a buy in: The player "benjam @ laRAysvKgm" quits the game with a stack of 0.00.
```
Comment (blerud) [inline ext/pokernow.py:76]: ???

File: ext/pokernow.py [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+import logging
+import seaborn as sns
+import matplotlib.pyplot as plt
+
+from io import StringIO, BytesIO
+from collections import defaultdict
+from discord import File
+from discord.ext import commands
+from discord.ext.commands import Context
+# from util.config import POKERNOW_LOG_DIR
```
Comment (blerud) [inline ext/pokernow.py:11]: ???

File: util/config.py (hunk 2 above)
Comment (blerud) [reply util/config.py:7]: and this is unused

**Comment by blerud** [comment]
failed linting fix ur shit

**Comment by blerud** [comment]
also u didnt add ext/pokernow to the extensions list so it wont load

File: ext/pokernow.py (hunk 4 above)
Comment (Frank5000) [reply ext/pokernow.py:11]: TODO

File: ext/maga.py (hunk 1 above)
Comment (Frank5000) [reply ext/maga.py:1]: unused
